src/rebuild/scan.c

```diff
@@ -776,6 +776,14 @@ rebuild_tgt_scan_handler(crt_rpc_t *rpc)
 	/* check if the rebuild is already started */
 	rpt = rpt_lookup(rsi->rsi_pool_uuid, rsi->rsi_rebuild_ver);
 	if (rpt != NULL) {
+		if (rpt->rt_global_done) {
+			D_WARN("the previous rebuild "DF_UUID"/%d"
+			       " is not cleanup yet\n",
+			       DP_UUID(rsi->rsi_pool_uuid),
+		               rsi->rsi_rebuild_ver);
```

Comment:
(style)  code indent should use tabs where possible

Reply:
Bottom line is missing a tab to match others.